check_openapi.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Copyright 2020 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo "Testing OpenAPI specifications file"
+# shellcheck disable=2181
+
+if docker run --rm -v "${PWD}":/local/:Z openapitools/openapi-generator-cli validate -i ./local/openapi.json; then
```

Comment:
Have you seen that `openapi-generator-cli` tool is available in Pypi? Maybe it will be easy to automate using a "pip install" than running a container

Reply:
That's interesting, I will give it a try.

Reply:
@JoseLSegura 
> This project checks the maven repository once a day for a new version and will publish this new version automatically as a python package.

I'm afraid it still requires java :)

Reply:
Dammit Java!!!